### ISSUE TITLE
Simple Div component that extends BaseDominoElement

### ISSUE THREAD

**Issue by masterdany88** [DominoKit/domino-ui#487]
Using elemento div() method 
```
    public static HtmlContentBuilder<HTMLDivElement> div() {
        return htmlElement("div", HTMLDivElement.class);
    }
```
 from `org.jboss.elemento;Elements` is quite strange. It is hard to read and write. It return builder class, or plaint  `HTMLDivElement` which lack of many usefull feature.

Domino UI introduce many good feature in BaseDominoElement feature. It is nice to work with it.
Sometimes I just need plaint div element, so it could be good to take it from UI library (domino uI) instead of elemento or elemental.

The most common way for me now is to use Row component, but it often offer to much feature.

So basically I need some class to create div from BaseDominoElement, that can be initialized and avoid using elemento/elemental.

**Comment by vegegoku** [comment]
You can create a div element that is a DominoElement using `DominoElement.div()` and for any other element you can just wrap any element or elemento element as a DominoElement using `DominoElement.of(...)`

You can also extend BaseDominoElement ..check the domino-ui components to see how we do this.

**Comment by masterdany88** [comment]
Nice. It id very useful
